### DISCUSSION TITLE
How to build VoxelFile with Tiles from Minecraft chunks?

### DISCUSSION THREAD

**Discussion by Steve3341** [ppiastucki/recast4j#167] [General]
Help me please 
[snippet.txt](https://github.com/ppiastucki/recast4j/files/13752215/snippet.txt)

**Comment by ppiastucki** [comment]
I am not familiar with Minecraft chunks, but you can add a single span to the heightfield by calling:
```
RecastRasterization.addSpan()
```
See:
https://github.com/ppiastucki/recast4j/blob/main/recast/src/main/java/org/recast4j/recast/RecastRasterization.java#L67

**Reply by Steve3341** [reply]
why no tiles and split 
![image](https://github.com/ppiastucki/recast4j/assets/154595170/65e8d307-dc69-4804-a052-98733099a6e4)
![image](https://github.com/ppiastucki/recast4j/assets/154595170/a91153a1-353e-48e1-b98a-345307cb0e4f)

**Reply by Steve3341** [reply]
code

[code.txt](https://github.com/ppiastucki/recast4j/files/13757015/code.txt)

**Comment by Steve3341** [comment]
i need like this test_tiles.voxels
![image](https://github.com/ppiastucki/recast4j/assets/154595170/5d6b982c-3d2e-4549-b644-8dcedad4dceb)

**Comment by Steve3341** [comment]
solved
